contributors/workflow.md

```diff
@@ -60,7 +60,7 @@ building, doc generation, etc:
 
 ```sh
 cd $working_dir/tidb/
-ln -s hooks/pre-commit .git/hooks/
+ln -s ../../hooks/pre-commit .git/hooks/
```

Comment:
`hooks/pre-commit` is located in the tidb source code directory, not in `../../`

Reply:
Yes, but the real path of `pre-commit` is `.git/hooks/../../hooks/pre-commit`. By the way, this is easy to test in the local environment.

Reply:
gotcha